core/invoker/src/main/scala/whisk/core/containerpool/kubernetes/KubernetesContainer.scala

```diff
@@ -138,4 +139,9 @@ class KubernetesContainer(protected[core] val id: ContainerId,
       .takeWithin(waitForLogs)
       .map { _.toByteString }
   }
+
+  /** Rename container. */
+  override def rename(name: String)(implicit transid: TransactionId): Future[Unit] =
+    // rename currently not supported
+    Future.successful(Unit)
```

Comment:
Would be easy enough to implement rename by making a call to change the value of the `name` label of the container.  However, this will add some latency, so as above we should decide if we can do it asynchronously and ignore any errors or synchronously so errors can be cleanly detected and handled.

Reply:
From my brief observations of invoker logs, after running system tests:
rename latency - average 43msec
initialization latency - average 185msec
https://gist.github.com/kpavel/a424db24139d83075abb31b0de365885
https://gist.github.com/kpavel/3464751e26b600a087b603c4767b3fa4

Although rename latency is lower than initialization one, it is too significant to ignore. I think it would be better to make it async instead of increasing initialization latency by 25%